packages/web-react/src/components/Grid/useGridStyleProps.ts

```diff
@@ -31,22 +46,31 @@ export function useGridStyleProps(props: SpiritGridProps<ElementType>): GridStyl
     ...useSpacingStyle(spacingY, 'grid', DirectionAxis.Y),
   };
 
-  let classes: string;
-  let gridColsClass: string;
+  function generateGridClass(componentClass: string, property: AlignmentOrCols, type: string) {
```

Comment:
Is there any reason why it's not via const and arrow function?

Reply:
I think it is due to be consistent with the rest of the file. 🤷

Reply:
> I think it is due to be consistent with the rest of the file. 🤷

Yes, I'm not sure if we have any internal rules for arrow functions versus regular functions in the code.

Reply:
No, there is no such rule, but we mostly use the arrow function. The main difference is in scoping and use of `this` between the arrow function and the named function. We can discuss this later and maybe lay down some rules. But in this case, there is no difference. So I suggest letting this as it is and deciding and refactoring later.